src/makeInfinityAdder.js

```diff
@@ -1,10 +1,38 @@
 'use strict';
 
+const { run } = require("jest");
+
 /***
  * @return {function}
  */
 function makeInfinityAdder() {
   // write code here
+  const adder = (a) => {
+    let count = 0;
+    let total = a;
+
+    if(count === 0 && a === undefined){
+      return 0;
+    }
```

Comment:
The check for `count === 0` is redundant since `count` is always set to 0 when `adder` is called. Also, returning 0 immediately if `a` is `undefined` will prevent the function from summing subsequent calls.